discovery-infra/start_discovery.py

```diff
@@ -406,7 +406,10 @@ def nodes_flow(client, cluster_name, cluster):
                 ],
             )
 
-            if args.vip_dhcp_allocation:
+            if args.master_count == 1 and machine_net.has_ip_v6 and not machine_net.has_ip_v4:
+                set_cluster_machine_cidr(client, cluster.id, machine_net, set_vip_dhcp_allocation=False)
+                tf.change_variables({"libvirt_master_ips": [[helper_cluster.Cluster.get_ip_for_single_node_v6(client, cluster.id, args.vm_network_cidr6)]]})
```

Comment:
This line will break static IPs as it tries to update TF variables. Currently this causes an attempt to set a MAC address, which fails.

Reply:
I believe `if not args.with_static_ips:` can solve it.